WORKING_GROUPS.md

```diff
@@ -0,0 +1,203 @@
+# IPFS Project Working Groups
+
+## Index
+
+- [Description](#description)
+- [Working Group Lifecycle](#working-group-lifecycle)
+- [Current Working Groups](#current-working-groups)
+	- [Development and Maintenance Working Groups](#development-and-maintenance-working-groups)
+	- [Community, Education and Usage  Working Groups](#community-education-and-usage-working-groups)
+	- [Ongoing Efforts to Support Specific Uses](#ongoing-efforts-to-support-specific-uses)
+- [Other not yet classified as a working group](#other-not-yet-classified-as-a-working-group)
+
+
+## Description
+
+The IPFS Project Working Groups are teams of people that are appointed to research, develop and deploy work under the scope that the working group is focused. This structure is designed to provide clarity and direction to the project, enabling individual contributors to focus their time and energy on the areas they are most interested.
+
+Each Working Group has its own cadence, lifecycle and objectives.
+
+## Working Group Lifecycle
+
+TODO: Define how a working group gets created, evaled and how to decide if its purpuse has been fulfilled and should be dismantled
+
+## Current Working Groups
+
+### Development and Maintenance Working Groups
+
+- [Release](#release)
+- [Specifications](#specifications)
+- [Files (Unixfs), DEX and Data Structures on top of IPLD](#files-unixfs-dex-and-data-structures-on-top-of-ipld)
+- [Runtime Support and Build](#runtime-support-and-build)
+- [QA, Testing and Dev Team Enablement](#qa-testing-and-dev-team-enablement)
+- [Security](#security)
+- [Performance and Benchmarking](#performance-and-benchmarking)
+
+#### Release
+
+The Release Working Group manages the releases for go-ipfs, js-ipfs and its internal dependencies.
+
+Responsibilities include:
+
+- Structure the released process.
+- Define the release Roadmap.
+- Decide what gets into each release.
+- Test releases.
+- Announce and gather feedback from the community on each release.
+
+#### Specifications
```

Comment:
Having a generic "Specifications" WG invites the same kind of confusion we have with a generic "javascript team". People work on the specs that are relevant to their work and don't necessarily need to be involved in other specs. Do you intend to have a small "specifications" WG that deals with all specs or is this a large group that includes everyone who has these kinds of responsibilities for any spec.

Reply:
Did you get to see my response to your last comment on the Specifications Working Group? Here https://github.com/ipfs/ipfs/pull/285#discussion_r153808334

Reply:
Reposting it here to make it easier to find:

>  In the same way that the Internationalization Working Group participants are not the ones writing the translations to every language but rather create the platform and coordinate multiple contributors and possible vendors for those translations to exist, the Specifications working group should be more focused on ensuring that specs are written, clear, and that individuals that participate in other working groups and/or projects can effectively contribute to specs (through training, mentoring and so on).
> 
> The Specifications working needs to have more of a role of an Editor + Reviewer, create a Peer Review system and structure the spec writing process (templates, coordination, timelines and objectives).